### DISCUSSION TITLE
Can I access the args fields globally from anywhere in the project?

### DISCUSSION THREAD

**Discussion by hadishahpuri** [rebing/graphql-laravel#1097] [Ideas]
As the Args field parameters are available inside the HTTP Request object,
Is it possible to access these parameters from anywhere in the project?

Something like: GraphQL::args()

**Comment by mfn** [comment]
These args are only passed to the root query/mutations resolver.

In the distant past every resolver received those args but that was a major bug because individual fields can have their own args.

If you need them outside your root query/mutation which received the initial request, you have to come up with your own thing (e.g. getting the current `Request` object and extract it manually from there or so).

Maybe I can give a better advise if I could better understand the goal.

**Comment by hadishahpuri** [comment]
Thanks I got the answer I was lookig for :)
